### ISSUE TITLE
[4.0][a11y] Mark the link group in the Quickicon module as a list

### ISSUE THREAD

**Issue by zwiastunsw** [joomla/joomla-cms#20995]
Quick icon module include a set of navigation links. 
A group of links in the Quick icon module is not marked as a list. 
Links are not part of the list.
In addition, it shall be marked as a navigation element (nav).
See: [Info and Relationships (Level A)](https://www.w3.org/WAI/WCAG21/quickref/?versions=2.0&showtechniques=111#info-and-relationships)

**Comment by brianteeman** [comment]
Sorry but I haven't got a clue what you mean. You really are going to have to provide more information

**Comment by zwiastunsw** [comment]
Currently, the module contains a group of 6 links. But this is not a list of links. Each is covered by the div tag.

**Comment by brianteeman** [comment]
and why should it be a list of links?

If you want to say something is wrong and expect someone to fix it then you really need to explain what is wrong and why

**Comment by zwiastunsw** [comment]
We have here a set of navigation links. It doesn't matter that they look like buttons. This is a disordered list. And so it should be marked. And since this is a navigation shortcut set, you should assign a role=navigation to this set.

**Comment by brianteeman** [comment]
so how are these links any different to the links in the latest articles module. what is it that makes these links to be navigation links and the others are not.

**Comment by zwiastunsw** [comment]
Of course, not every set of links should be marked as a navigation element, **but every important navigation element - yes**. Bruce Lawson wrote about that in his book "Introduction HTML.5". In one of his presentations, however, he stressed that these must be only important sets of links. Because if everything is highlighted, nothing is highlighted. 
The links in the Quickicon module are an important navigation element on the Control Panel page because they lead to the most frequently used backend pages.  For this reason, they were placed as one of the first elements on the website and graphically highlighted.

**Comment by brianteeman** [comment]
the last two (update) and are more than just navigation

**Comment by zwiastunsw** [comment]
I agree, but they are also navigation.

**Comment by brianteeman** [comment]
closed - please test #22614
